src/main/java/org/hydev/mcpm/client/Launcher.java

```diff
@@ -7,9 +7,4 @@
  * @since 2022-09-27
  */
 public class Launcher
-{
-    public static void main(String[] args)
-    {
-        System.out.println("Meow");
-    }
-}
+{ public static void main(String[] args) {System.out.println("Meow");}}
```

Comment:
why is this file changed lol

Reply:
I just wanna borrow it for testing lmao and happen to change it hehe

Reply:
I'll fix this on main